src/components/state-lookup.ts

```diff
@@ -48,7 +49,7 @@ const DEFAULT_STATE_CONCURRENCY = 4;
 
 export class StateLookup {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    private _client: any;
+    private _client: any|Intent;
```

Comment:
what does this do? is there any value to `any|*`?

Reply:
Not a whole lot, just helpful for future reference. Annoyingly we don't have full types for matrix-js-sdk yet, so it's called `any` all over the place.